### DISCUSSION TITLE
Announcement of results

### DISCUSSION THREAD

**Discussion by ambros-gleixner** [ambros-gleixner/MIPcc23#25] [Announcements]
*Dear all,*

The final results have been announced and awarded at the 20th Anniversary MIP Workshop last week. Let me share them with you here.

The **winner** of the competition is [Krunal Patel](https://www.krooonal.com/) with his submission "Progressively Strengthening and Tuning MIP Solvers for Reoptimization":

*The submission convinced the jury not only through its computational excellence that was displayed by the top ranked performance on almost all public and hidden data sets, but also through its broad applicability and attention to algorithmic details. Building on top of an existing LP-based branch-and-bound solver, the approach distinguishes itself by targeting multiple aspects of the solving process in combination. The reuse of primal information and pseudo costs from previous solving processes is safeguarded by clever methods to re-evaluate and complete this information. By analyzing the success rates of individual solving techniques, the approach learns improved parameter configurations online despite the limited number of observations.*

An **honorable mention** has been awarded to the team of *Paul Strang, Zacharie Ales, Come Bissuel, Olivier Juan, Safia Kedad-Sidhoum, and Emmanuel Rachelson* for their submission "Influence branching for learning to solve mixed integer programs online":

*The jury was intrigued by the successful adaptation of influence branching, an existing graph-based branching strategy for MIP, to the reoptimization setting of the competition. The submission employs online hyperparameter tuning of different influence models via multi-armed bandit selection, a widely recognized online learning framework. Notably, the resulting method consistently performed well on both public and hidden datasets, showcasing its general effectiveness and potential impact.*

**Congratulations from the entire organizing committee!**
*Ambros Gleixner, Timo Berthold, Mathieu Besançon, Suresh Bolusani, Claudia D'Ambrosio, Gonzalo Muñoz, Joseph Paat, Domenico Salvagnin, and Dimitri Thomopulos*

**Comment by Ishaanjolly** [comment]
Congrats again Krunal!
